Easydict/App/Localizable.xcstrings

```diff
@@ -1405,7 +1405,7 @@
         "en" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Hide Menu Bar Icon"
+            "value" : "Hide menubar icon"
```

Comment:
Since Apple use `MenuBarExtra`, means it is menu bar, we can follow.

```swift
struct MenuBarExtra<Label, Content> where Label : View, Content : View
```

https://developer.apple.com/documentation/swiftui/menubarextra/